packages/aila/src/protocol/schema.ts

```diff
@@ -1,79 +1,75 @@
+import dedent from "dedent";
 import z from "zod";
 import { zodToJsonSchema } from "zod-to-json-schema";
 
+import { minMaxText } from "./schemaHelpers";
+
+// ********** BASED_ON **********
+export const BASED_ON_DESCRIPTIONS = {
+  id: dedent`The database ID of an existing lesson plan that this
+    lesson plan is based upon.
+    It should be the ID of the lesson plan that was used to inform
+    the content of this lesson plan.
```

Comment:
I think that dedent will maintain the line breaks, so there will be a break between "inform" and "the" in the output. Is that intentional?

Reply:
This applies to other uses of dedent as well

Reply:
Ah, good catch, thanks!